src/backend/metal/src/device.rs

```diff
@@ -703,6 +703,49 @@ impl Device {
 
         arg
     }
+
+    #[cfg(feature = "core_video")]
+    pub unsafe fn create_image_view_with_metal_texture(
+        &self,
+        texture: metal::Texture,
+        kind: image::ViewKind,
+        format: format::Format,
+        swizzle: format::Swizzle,
+        range: image::SubresourceRange,
+    ) -> Result<n::ImageView, image::ViewError> {
```

Comment:
is this mostly the same code as the device has for regular view creation? can one use another internally?

Reply:
The texture returned by `CVMetalTextureGetTexture()` isn't a framebuffer-only texture and I think it's alright to call '.to_owned()'. I should give it a try.